wireless/bluetooth/nimble/Kconfig

```diff
@@ -7,13 +7,221 @@ config NIMBLE
 		host-layer stack.
 
 if NIMBLE
-config NIMBLE_STACKSIZE
-	int "nimble stack size"
-	default DEFAULT_TASK_STACKSIZE
 
 config NIMBLE_REF
 	string "Version"
 	default "bc7828341226d860429c63994065f8f1b8b8d7b0"
 	---help---
 		Git ref name to use when downloading from nimBLE repo
-endif
+
+if DEBUG_FEATURES
+
+config NIMBLE_DEBUG_ERROR
+	bool "Apache nimBLE error output"
+	default n
+	depends on DEBUG_ERROR
+
+config NIMBLE_DEBUG_WARN
+	bool "Apache nimBLE warnings output"
+	default n
+	depends on DEBUG_ERROR
+
+config NIMBLE_DEBUG_INFO
+	bool "Apache nimBLE informational debug output"
+	default n
+	depends on DEBUG_INFO
+
+endif # DEBUG_FEATURES
+
+config NIMBLE_TINYCRYPT
+	bool "Apache nimBLE tinycrypt support"
+	default y
+
+config NIMBLE_MESH
+	bool "Apache nimBLE mesh support"
+	default n
+	depends on EXPERIMENTAL
+
+menu "NimBLE role configuration"
+
+config NIMBLE_ROLE_BROADCASTER
+	bool "nimBLE broadcaster role"
+	default n
+
+config NIMBLE_ROLE_CENTRAL
+	bool "nimBLE central role"
+	default n
+
+config NIMBLE_ROLE_OBSERVER
+	bool "nimBLE observer role"
+	default n
+
+config NIMBLE_ROLE_PERIPHERAL
+	bool "nimBLE peripheral role"
+	default n
+
+endmenu # "NimBLE role configuration"
+
+choice
+	prompt "NimBLE BLE version"
+	default NIMBLE_BLE_VERSION_50
+
+config NIMBLE_BLE_VERSION_50
+	bool "NimBLE BLE version 5.0"
```

Comment:
Can we use `nimBLE` or `NimBLE` in all the places?

Reply:
Done. I chose `NimBLE` as it is in the mynewt-nimble repo